apps/backend/src/poll/poll/poll.service.ts

```diff
@@ -235,7 +235,7 @@ export class PollService implements OnModuleInit {
       if (!oldEvent) {
         return false;
       }
-      return oldEvent.start.valueOf() !== event.start.valueOf() || oldEvent.end.valueOf() !== event.end.valueOf();
+      return oldEvent.start.valueOf() !== new Date(event.start).valueOf() || oldEvent.end.valueOf() !== new Date(event.end).valueOf();
```

Comment:
Why is `new Date` necessary? `event.start` and `.end` are `Date` objects due to `@Transform`

Reply:
Fun fact: That doesn't seem to be the case. I didn't look deeply into this, but I'll have a look into the Transform pipe again. At this stage of code the event.start was of type `string` instead of `Date`, which led to the comparison of e.g. `17231200` and `2024-bla-bla`.